bigtable/docs/snippets.py

```diff
@@ -430,12 +426,6 @@ def test_bigtable_list_tables():
     tables_list = instance.list_tables()
     # [END bigtable_list_tables]
 
-    table_names = [table.name for table in tables_list]
-    try:
-        assert table.name in table_names
-    finally:
-        retry_429(table.delete)()
-
```

Comment:
Removing the table creation here also removes the value of the snippet as a test.  Rather, we should invoke `table.create()` with a longer timeout (the [configured timeout](https://github.com/googleapis/google-cloud-python/blob/6fc0de5bc489394a4c88f35325abed18fa5e6cb9/bigtable/google/cloud/bigtable_admin_v2/gapic/bigtable_table_admin_client_config.py#L48) is 130 seconds).

Reply:
That is good catch. My bad, I missed the assert.
Added the assert back.

Reply:
As we are using existing table now, we don't need table.create() here.